src/kakarot/instructions/block_information.cairo

```diff
@@ -71,4 +75,37 @@ namespace BlockInformation {
         let ctx = ExecutionContext.increment_gas_used(ctx, GAS_COST_COINBASE);
         return ctx;
     }
+
+    // @notice NUMBER operation.
+    // @dev Get the block number
+    // @custom:since Frontier
+    // @custom:group Block Information
+    // @custom:gas 2
+    // @custom:stack_consumed_elements 0
+    // @custom:stack_produced_elements 1
+    // @return The pointer to the updated execution context.
+    func exec_number{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr}(
+        ctx: model.ExecutionContext*
+    ) -> model.ExecutionContext* {
+        %{ print("0x43 - NUMBER") %}
+        // Get the block number.
+        // let current_block : felt = 1000;
+        let (current_block) = get_block_number();
```

Comment:
Please remove commented lines and the print

Reply:
Removed :)